runtime/src/iree-amd-aie/driver/hsa/cts/CMakeLists.txt

```
@@ -0,0 +1,103 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(CMakeDependentOption)
+
+set(PEANO_INSTALL_DIR "" CACHE PATH "")
+set(VITIS_DIR "" CACHE PATH "")
+if((NOT PEANO_INSTALL_DIR) AND (NOT VITIS_DIR))
+  message(FATAL_ERROR "either PEANO_INSTALL_DIR or VITIS_DIR must be set")
+endif()
+cmake_dependent_option(USE_CHESS "" "1" "VITIS_DIR" "0")
+set(TARGET_DEVICE "npu1_4col" CACHE STRING "")
+
+iree_bytecode_module(
+  NAME
+    amd-aie-hsa_executable_cache_test_module
+  MODULE_FILE_NAME
+    amd-aie-hsa_executable_cache_test.bin
+  SRC
+    "${CMAKE_CURRENT_LIST_DIR}/executable_cache_test.mlir"
+  FLAGS
+    --compile-mode=hal-executable
+    --iree-hal-dump-executable-files-to=${CMAKE_CURRENT_BINARY_DIR}
+    --iree-hal-target-backends=amd-aie-hsa
+    --iree-amdaie-lower-to-aie-pipeline=air
```

This should work with objectFifo as well? If so, we should use that as the default.
```suggestion
    --iree-amdaie-lower-to-aie-pipeline=objectFifo
```

---

Doesn't work: I get

```
[AIE ERROR] _XAie_LoadProgMemSection():231: Overflow of program memory
```

---

Ok, interesting. I guess keep it on air then for now.